style.css

```diff
@@ -1 +1,2 @@
 .btn { color:blueviolet; }
```

Comment:
:red_circle: Kindly fix the Stylelint error in this file. To fix the error Kindly You can read [this comment](https://github.com/david30-maker/Hello-World/pull/1#issuecomment-1377891799)